packages/devtools_app/lib/src/profiler/flutter/cpu_profiler.dart

```diff
@@ -22,42 +23,90 @@ class CpuProfiler extends StatelessWidget {
 
   final Function(CpuStackFrame stackFrame) onStackFrameSelected;
 
+  static const Key expandButton = Key('CpuProfiler - Expand Button');
+  static const Key collapseButton = Key('CpuProfiler - Collapse Button');
```

Comment:
nit: r/expandButton/expandButtonKey ? this seems to be the naming pattern we use elsewhere

Reply:
Done.